ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -159,15 +159,15 @@
    * to complete.
    */
   @Config("store.stats.wait.timeout.in.secs")
-  @Default("60")
+  @Default("2*60")
   public final long storeStatsWaitTimeoutInSecs;
 
   /**
-   * Specifies the number of index segments that can be read per second for stats collection.
+   * Specifies the number of index entries that can be read per second for stats collection.
    */
-  @Config("store.stats.index.segment.per.secs")
-  @Default("1")
-  public final int storeStatsIndexSegmentPerSecs;
+  @Config("store.stats.index.entries.per.second")
+  @Default("2400000")
```

Comment:
I don't remember the exact numbers, but we had a conversation on this and after some calculations we agreed upon on some default value that would ensure the scan to finish in less than a hour.